### PR TITLE
FEAT: custom autoscaling stats

### PR DESCRIPTION
**Pull request by aaronkvanmeerten**

### CODE REVIEW

File: src/autoscaler.ts [hunk 1]
```diff
@@ -14,6 +15,29 @@ export interface AutoscaleProcessorOptions {
     redisClient: Redis.Redis;
 }
 
+const groupsManaged = new promClient.Gauge({
+    name: 'autoscaling_groups_managed',
```
Comment (jmacelroy) [inline src/autoscaler.ts:19]: Will these names cause collisions in the future or do labels take care of it? I feel like even with labels we may want to say "jibri_autoscaling_groups_managed" or something more specific.

File: src/instance_launcher.ts [hunk 2]
```diff
@@ -63,23 +88,37 @@ export default class InstanceLauncher {
...
-        } else if (count > group.scalingOptions.desiredCount && count > group.scalingOptions.minDesired) {
-            ctx.logger.info('[Launcher] Will scale down to the desired count', { groupName, desiredCount, count });
+                // increment launched instance stats for the group
+                instancesLaunched.inc({ group: group.name }, actualScaleUpQuantity);
+            } else if (count > group.scalingOptions.desiredCount && count > group.scalingOptions.minDesired) {
+                ctx.logger.info('[Launcher] Will scale down to the desired count', { groupName, desiredCount, count });
 
-            const actualScaleDownQuantity =
-                count - Math.max(group.scalingOptions.minDesired, group.scalingOptions.desiredCount);
+                const actualScaleDownQuantity =
```
Comment (jmacelroy) [inline src/instance_launcher.ts:106]: Are these unintended whitespace changes?

Comment (aaronkvanmeerten) [reply src/instance_launcher.ts:106]: I moved this whole block into a try {} to catch the errors by group, so this was intentional.

File: src/autoscaler.ts (hunk 1 above)
Comment (aaronkvanmeerten) [reply src/autoscaler.ts:19]: This is the total number of groups being managed by the service regardless of type.  We may have another stat which has the label of 'type' which could show the count of jibri groups vs. JVB groups or jigasi groups, etc.

**Pull request merged**